Server/ConcordiaCurriculumManager/appsettings.Development.json

```diff
@@ -21,15 +21,34 @@
   "SenderEmailSettings": {
     "SenderSMTPHost": "smtp.gmail.com",
     "SenderSMTPPort": "587",
-    "SenderEmail": "test@gmail.com",
-    "SenderPassword": "not a valid password"
+    "SenderEmail": "ccmc9531@gmail.com",
+    "SenderPassword": "fgym vpwn eths umii"
   },
   "Serilog": {
+    "MinimumLevel": {
+      "Default": "Information",
+      "Override": {
+        "Microsoft": "Warning"
+      }
+    },
     "WriteTo": [
       {
-        "Name": "Console",
+        "Name": "PostgreSQL",
```

Comment:
Is there a reason you change all this config here?

Reply:
I didn't change that